utils.py

```diff
@@ -65,9 +65,11 @@ def load_cfg(yaml_filepath):
         cfg : dict
     """
     logger.info("Loading experiment yaml file")
-    # Read YAML experiment definition file
+
+    yaml=YAML(typ='safe')
     with open(yaml_filepath, "r") as stream:
-        cfg = yaml.load(stream, Loader=yaml.FullLoader)
+        # This will fail if there are duplicated keys in the YAML file
```

Comment:
So, why not add a try/except?

Reply:
We (I) want it to fail quickly if there is a duplicated key because the only solution is for the user to modify the config file.